ddp-workspace/projects/ddp-osteo/src/app/app.module.ts

```diff
@@ -63,6 +63,7 @@ tkCfg.parentalConsentUrl = 'parental-consent';
 tkCfg.releaseMinorUrl = 'release-minor-survey';
 tkCfg.releaseUrl = 'release-survey';
 tkCfg.dashboardUrl = 'dashboard';
+tkCfg.doneUrl = 'proxy-thank-you';
```

Comment:
You sure about you want the DONE workflow state to always be mapped to going to the Age Up thank you? (At least that is what it looks like Osteo is now configured to do.

Reply:
Yes, I discussed it with @yufengwng

Reply:
Yup, that's the current plan for Osteo's configuration. Better ideas here are welcomed!